src/obfl-specification.html

```diff
@@ -1195,6 +1200,72 @@ <h4 id="L722" class="include">evaluate</h4>
           <dd class="markup note">This attribute is only available in the context of a field element.</dd>
       </dl>
     </dd>
+  <dt>Variables</dt>
+    <dd class="markup note">For readability, the examples below have plain text
+      instead of braille.</dd>
+    <dd>
+      <p>These are the commonly available variables for use in the <a
+        href="obfl-evaluation-language.html">OBFL Evaluation Language</a>
+        expression:</p>
+      <dl>
+        <dt>page</dt>
+          <dd>The variable holding the current page number. For instance,
+            <code>This is page &lt;evaluate expression="$page"/&gt;</code>.</dd>
+        <dt>volume</dt>
+          <dd>The variable holding the current volume number. For instance,
+            <code>This is volume &lt;evaluate
+            expression="$volume"/&gt;</code>.</dd>
+        <dt>volumes</dt>
+          <dd>The variable holding the total number of volumes. For instance,
+            <code>This book has &lt;evaluate expression="$volumes"/&gt;
+              volumes</code>.</dd>
+        <dt>sheets-in-document</dt>
+          <dd>The variable holding the number of sheets in this document. For
+            instance, <code>This document has &lt;evaluate
+            expression="$sheets-in-document"/&gt; sheets</code>.</dd>
+        <dt>sheets-in-volume</dt>
+          <dd>The variable holding the number of sheets of the current volume.
+            For instance, <code>This volume has &lt;evaluate
+            expression="$sheets-in-volume"/&gt; sheets</code>.</dd>
+      </dl>
+      <p>These common variables can also be used in all use-when conditions.</p>
+    </dd>
+    <dd>
+      <p class="markup">This variable is available in descendant elements of
+        on-volume-start and on-volume-end:</p>
```

Comment:
The link is automatically generated because of the `class="markup"`. Please have a look at this temporary available [preview](https://rambags.home.xs4all.nl/obfl/obfl-specification.html#L722).